synapse/cores/common.py

```diff
@@ -1559,8 +1563,8 @@ def splice(self, mesg):
             core.splice(mesg)
 
         '''
-        act = mesg[1].get('act')
-        self.spliceact.react(mesg, name=act)
+        data = mesg[1]['mesg']
```

Comment:
id do something like ``act, _mesg = mesg[1].get('mesg')`` then pass those to the reactor as ``self.spliceact.react(mesg, act)``

Reply:
changed